opengraph/dom-test/conftest.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import pytest
+from bs4 import BeautifulSoup
+
+
+# Load url and parse it
+@pytest.fixture
+def meta(response_obj):
+    soup = BeautifulSoup(response_obj.content, "html.parser")
+    return soup.find_all("meta")
+
+
+@pytest.fixture
+def response_obj():
+    return requests.get('http://nightly.mozilla.org')
```

Comment:
Per our conversation, you'll need to pull in and loop through a list of all the sites you want to test (not just this one URL).  That list should ideally reside in a config file instead.